### PR TITLE
Update to new build process and turn on lint.

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

File: adafruit_ht16k33/ht16k33.py [hunk 1]
```diff
@@ -24,6 +51,7 @@ def _write_cmd(self, byte):
             self.i2c_device.write(self._temp)
 
     def blink_rate(self, rate=None):
```
Comment (dhalbert) [inline adafruit_ht16k33/ht16k33.py:53]: This could be a get/set property.

File: adafruit_ht16k33/ht16k33.py [hunk 2]
```diff
@@ -32,19 +60,22 @@ def blink_rate(self, rate=None):
                         _HT16K33_BLINK_DISPLAYON | rate << 1)
 
     def brightness(self, brightness):
```
Comment (dhalbert) [inline adafruit_ht16k33/ht16k33.py]: This could be a get/set property.

**Review by dhalbert — COMMENTED** [review]
Maybe @deshipu would like to review also. Added him as as reviewer.

**Comment by tannewt** [comment]
This is for https://github.com/adafruit/circuitpython/issues/475

File: adafruit_ht16k33/ht16k33.py (hunk 1 above)
Comment (tannewt) [reply adafruit_ht16k33/ht16k33.py:53]: I know I filed a separate issue for it: #2

File: adafruit_ht16k33/ht16k33.py (hunk 2 above)
Comment (tannewt) [reply adafruit_ht16k33/ht16k33.py]: #2

File: README.rst [hunk 3]
```diff
@@ -1,8 +1,76 @@
...
 compatible with MicroPython machine API see this library: https://github.com/adafruit/micropython-adafruit-ht16k33
+
+Installation
+=============
+This driver depends on many other libraries! Please install it by downloading
+`the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
+
+Usage Example
+=============
+Using it is super simple. Simply import the `cpx` variable from the module
```
Comment (deshipu) [inline README.rst]: cpx variable?

File: adafruit_ht16k33/index.rst [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+.. automodule:: adafruit_ht16k33.ht16k33
+   :members:
+
+`matrix` - Matrix of dots in the displays
+-----------------------------------------------------
+
+.. automodule:: adafruit_ht16k33.matrix
+   :members:
+
+`segments` - Segments display such as those on alarm clocks
```
Comment (deshipu) [inline adafruit_ht16k33/index.rst]: Segment displays

File: conf.py [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'README'
+
+# General information about the project.
+project = u'Adafruit HT16K33 Library'
+copyright = u'2016 Radomir Dopieralski'
+author = u'Radomir Dopieralski'
```
Comment (deshipu) [inline conf.py:35]: I'm not the author of this library, or I must have been very drunk and don't remember it.

**Review by deshipu — CHANGES_REQUESTED** [review]

**Comment by deshipu** [comment]
Tested for the 14-seg display, the `segments.py` example works after commenting out `display = segments.Seg7x4(i2c)` and uncommenting `#display = segments.Seg14x4(i2c)`.

**Pull request merged**